docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
 Column-based Noise
 ==================
 
+Column-based noise introduces errors in individual fields within a row of data.
+The types implemented in pseudopeople are listed below, along with instructions
+about how to configure them.
+
+.. contents::
+   :depth: 2
+   :local:
+
 Leave a field blank
 -------------------
 
+When this noise type occurs, whatever value was supposed to be collected is missing for this row.
```

Comment:
```suggestion
When this noise type occurs, whatever value was supposed to be collected is missing for this cell.
```

Reply:
It would also be good if we could somehow clarify that "whatever value was supposed to be collected" is the column name, i.e., drive home the point that we are applying this to a specific column. With the current description, it seems like we could be blanking out cells willy-nilly all over the dataframe.

Reply:
Here's a suggested edit for this whole paragraph, switching the order around:

> Often some of the data in certain columns of a dataset will be missing. This could be because the input for that field was left blank, an answer was refused, or the answer was illegible or unintelligible. When this type of noise occurs, pseudopeople will replace the value in the relevant cell with ``numpy.nan`` to indicate that the value is missing.